openlibrary/i18n/validators.py

```diff
@@ -0,0 +1,44 @@
+from babel.messages.catalog import TranslationError
+from babel.messages.checkers import python_format
+
+
+def validate(message, catalog):
+    errors = _validate_fuzzy(message)
+    errors.extend(_validate_format(message, catalog))
+
+    return errors
+
+
+def _validate_format(message, catalog):
```

Comment:
If you can find the type of these inputs, adding would be nice. Nice-to-have, not a blocker.

```suggestion
def _validate_format(message: Message, catalog: Catalog) -> List[str]:
```